### PR TITLE
test: relocate test for redefining variable while mixing notations

### PR DESCRIPTION
**Pull request by galtm**
This minor test refactoring aims to keep similar things together. Fixes issue #927.

### CODE REVIEW

**Comment by AirQuick** [comment]
Regarding the pull request title, it's true that this pull request is a refactoring and the [type](https://github.com/xspec/xspec/blob/master/CONTRIBUTING.md#type) definition is ambiguous. But the `test` type would be better in this case, because this pull request does not change the primary code of XSpec. That is, `refactor: relocate test for redefining variable while mixing notations` .

File: test/variable.xspec [hunk 1]
```diff
@@ -173,6 +170,25 @@
 			<x:expect label="reflects value redefined within same scenario."
 				select="'value #2 (redefined using same prefix or different prefix)'"/>
 		</x:scenario>
+		<x:scenario label="Redefinition of variable and mixed notations for its name">
+			<x:variable as="element(variable-child)"
+				name="
+				&#x09;&#x0A;&#x0D;&#x20;
+				Q{http://example.org/ns/my/variable}var
+				&#x09;&#x0A;&#x0D;&#x20;"
```
Comment (AirQuick) [inline test/variable.xspec:178]: Whitespaces were inserted into `eqname.xspec` by #803.
Even after this test relocation, a local variable name with whitespaces is still covered by [`eqname.xspec`](https://github.com/galtm/xspec/blob/1f87de3b37ed5ef14620ff3201d7bb385f3dbdbe/test/eqname.xspec#L101-L104). So I think we can remove the whitespaces from `variable.xspec`:

```suggestion
				name="Q{http://example.org/ns/my/variable}var"
```

**Comment by galtm** [comment]
> the `test` type would be better in this case

No problem. I changed it.

File: test/variable.xspec (hunk 1 above)
Comment (galtm) [reply test/variable.xspec]: Fixed with df7e559203cf3a052059d8f6d6f91fa5f733e329

**Pull request merged**